src/SSVOpenHexagon/Online/Online.cpp

```diff
@@ -516,7 +516,7 @@ void initializeValidators(HGAssets& mAssets)
 
         HG_LO_VERBOSE("hg::Online::initializeValidators")
             << "Finished initializing validators...\n";
-    */
+    //*/
```

Comment:
```suggestion
    */
```

Reply:
Mistake?

Reply:
a hacky way of abusing multi-line comments. Very handy for testing and such, when you put another slash at the beginning of it while the end of the comment already has double slash - it will uncomment the thing and you don't have to get rid of the ending of it - `*/`
```cpp
/*
void main(){ return 0; }
//*/
```
```cpp
//*
void main(){ return 0; }
//*/
```

Reply:
Oh, but idk i didn't touch Online header though huh, well uh I thought that was my thing.